src/main/java/com/cloudbees/hudson/plugins/folder/computed/ComputedFolder.java

```diff
@@ -248,6 +248,16 @@ protected void submit(StaplerRequest req, StaplerResponse rsp) throws IOExceptio
         return triggers.toMap();
     }
 
+    public void addTrigger(Trigger trigger) {
+        Trigger old = triggers.get(trigger.getDescriptor());
+        if (old != null) {
+            old.stop();
+            triggers.remove(old);
+        }
+        triggers.add(trigger);
+        trigger.start(this, true);
```

Comment:
Should it be explicitly saved here?
/me wonders about the <code>@PersistsData</code> annotation